### ISSUE TITLE
Rendering issues with Entity:SetRenderClipPlane() after December Update

### ISSUE THREAD

**Issue by Artyymis** [Facepunch/garrysmod-issues#5152]
This function does not render clipped entities correctly.

System Specs

Motherboard: MSI x470 Gaming Plus Max
CPU: Ryzen 5 2600x
RAM: 2x Corsair Vengeance 8GB
GPU: MSI Radeon RX 6600XT

**Comment by FlorianLeChat** [comment]
Including additional information (version and branch of your game, reproduction code, ...) as normally requested by the GitHub template may be more helpful to Rubat and others to solve your issue.

**Comment by Artyymis** [comment]
Right, my apologies. My current game version is (2022.01.27). I was originally using the default branch of the game as I was completely unaware that there was a 64-bit branch of the game. But I was still having the issues, and after a friend suggested that I try using the "x86-64 - Chromium + 64-bit binaries" branch to see if the problem persisted. Which it did. Props and many other entities we not clipping properly.

**Comment by Artyymis** [comment]
Can anyone still possibly help me please?

**Comment by Kefta** [comment]
Do you have reproduction steps/code?

**Comment by Artyymis** [comment]
Unfortunately no, it just started happening after an update which I assume to be the december update. I've tried running every different branch of the game (64bit and all others). I've tried removing all mods, and just recently I've done a complete factory reset of my entire computer and the issue still persists. I really wish I could give you more information but I just have no idea what is the problem. A friend suggested it could be because I recently have upgraded my gpu from a RX 570 to a RX 6600XT? Still am very frustrated with it and have been trying to diagnose why it's happening or what on earth is causing models to not be clipped along the plane right.

**Comment by Kefta** [comment]
Screenshots at least?

**Comment by Artyymis** [comment]
Could I friend you on steam or something to make it easier?

**Comment by Artyymis** [comment]
Here is a gif of different models, I'm using the Carters Addon Pack which adds stargates and models are supposed to be clipped along the blue puddle (such as the dumpster model. However the other models show that not all entities or props get clipped properly. I hope this helps and if you have any suggestions for other ways to use (Entity:SetRenderClipPlane() ) Please let me know.

Link: https://imgur.com/a/tlh2HKo
